### PR TITLE
Update version to 1.6.0

### PR DESCRIPTION
**Pull request by duli2012**
**Description**: Describe your changes.
Update version to 1.6.0

### CODE REVIEW

File: docs/Versioning.md [hunk 1]
```diff
@@ -26,6 +26,7 @@ For more details on ONNX Release versions, see [this page](https://github.com/on
 
 | ONNX Runtime release version | ONNX release version | ONNX opset version | ONNX ML opset version | Supported ONNX IR version | [Windows ML Availability](https://docs.microsoft.com/en-us/windows/ai/windows-ml/release-notes/)|
 |------------------------------|--------------------|--------------------|----------------------|------------------|------------------|
+| 1.6.0 | **1.7** down to 1.2 | 12 | 2 | 6 | Windows AI 1.5+ |
```
Comment (faxu) [inline docs/Versioning.md:29]: should be opset 13

Comment (duli2012) [reply docs/Versioning.md]: fixed

**Review by pranavsharma — APPROVED** [review]

File: docs/Versioning.md (hunk 1 above)
Comment (faxu) [reply docs/Versioning.md]: one more update - the Windows AI 1.5+ should be 1.6+

**Review by faxu — APPROVED** [review]

**Review by pranavsharma — APPROVED** [review]

**Pull request merged**